### PR TITLE
Update docs to include the alias parameter.

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

File: index.mako [hunk 1]
```diff
@@ -164,6 +164,14 @@ repository's configuration.
 =r=
     =c= `id`
     =c= which hook from the repository to use.
+=r=
+    =c= `alias`
+    =c= (optional) Provide an alias to the hook. The `alias` allows
+        differentiation for when one wants to include the same hook more
+        than once, with, for example, different parameters. This allows
+        `pre-commit run <alias>` to target a specific entry of the hook to
+        run.
```
Comment (asottile) [inline index.mako]: I'd say something like

(optional) allows the hook to be referenced using an additional id when using `pre-commit run <hookid>`.

(this matches the capitalization and is brief and too the point)

Comment (s0undt3ch) [reply index.mako]: Updated

**Comment by asottile** [comment]
Thanks :tada:

**Comment by s0undt3ch** [comment]
Anytime!

**Pull request merged**